tasks-dev/assemble.js

```diff
@@ -3,9 +3,8 @@ const assemble = require('assemble');
 const helpers = require('handlebars-helpers');
 const debug = require('gulp-debug');
 const plumber = require('gulp-plumber');
-const newer = require('gulp-newer');
+// const newer = require('gulp-newer');
```

Comment:
Think this is in there as the newer part of the task is also commented out – may be something we use in the future if we manage to fix?